src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/table.scss

```diff
@@ -78,4 +108,50 @@ $cellBackgroundColorActive: $shakespeare;
         border-radius: 0 3px 3px 0;
         border-right-width: 1px;
     }
+
+    &.small {
+        width: 1%;
```

Comment:
why `1%` wouldn't `1px` be the better choice? (Would be smaller or equal than the actual width for sure)